StepDefinitions/LanguageTestsStepDefinitions.cs

```diff
@@ -0,0 +1,71 @@
+using MarsSpecFlow_HannahN.Pages;
+using MarsSpecFlow_HannahN.Utils;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsSpecFlow_HannahN.StepDefinitions
+{
+    [Binding]
+    internal class LanguageTestsStepDefinitions : CommonDriver 
+    {
+        LoginPage loginPageObj = new LoginPage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors